orchestra/frontend/src/global.d.ts

```diff
@@ -0,0 +1,6 @@
+interface Window {
+  orchestra?: {
+    tasks: React.FC<{ id: string }>;
```

Comment:
use a similar format to the Angular version
orchestra:
   workflows:
     workflow_slug:
         version_slug:
             step_slug:
                 Component

e.g.,
window.orchestra.workflows.example_workflow.v3.example_step = StepComponent